modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oidc/OIDCSSOConsentTestCase.java

```diff
@@ -75,6 +75,8 @@ public void testInit() throws Exception {
 
         initUser();
         createUser(user);
```

Comment:
If this created user is not used in this test case. Shall we remove that along with this fix?

Reply:
In the test case it asserts  `0_Email,1_First Name` for the consent. Therefore we need to login from the user created here in order to get the correct consent that tries to assert.